indico/modules/events/editing/notifications.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from flask import session
+
+from indico.core.notifications import email_sender, make_email
+from indico.web.flask.templating import get_template_module
+from indico.web.flask.util import url_for
+
+
+def _get_timeline_url(revision):
+    editable = revision.editable
+    return url_for('event_editing.editable', editable, _external=True)
+
+
+@email_sender
```

Comment:
I'm not a big fan of the decorator... I would just use `send_email(make_email(...))` in the function itself.